react/features/video-quality/components/VideoQualityDialog.web.js

```diff
@@ -281,7 +303,11 @@ class VideoQualityDialog extends Component {
      * @returns {void}
      */
     _enableLowDefinition() {
-        sendAnalyticsEvent(TOOLBAR_VIDEO_QUALITY_LOW);
+        // Note that this should only execute when the user of the application
+        // interacts with the quality selection UI. If in the future this
+        // executes for another reason (e.g. as a result of a call of the
+        // external API) the code must be refactored to not fire this event.
```

Comment:
I'm not certain these notes are necessary. These components should not be triggering UI-event callbacks based on another action. There would be bigger design issues if code were reaching into this dialog and acting on it.

Reply:
OK, removing